rvsol/src/RISCV.sol

```diff
@@ -285,34 +285,44 @@ contract RISCV is IBigStepper {
                 out := 0
             }
             function stateOffsetPreimageKey() -> out {
-                out := add(stateOffsetMemRoot(), stateSizeMemRoot())
+                out := 32 // 0 + 32
+                    //                out := add(stateOffsetMemRoot(), stateSizeMemRoot())
```

Comment:
Maybe instead we should make all of these constants?

Reply:
I don't think solidity yul supports constants in the language specification 😞 (I found this closed issue https://github.com/ethereum/solidity/issues/5928)